grpc/python_client/fly_mission.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+
+from __future__ import print_function
+import grpc
+from threading import Thread
+import time
+import dronecore_pb2 as dc
+import dronecore_pb2_grpc
+
+
+thread_status = True
+
+
+def wait_func(future_status):
+    global thread_status
+    ret = future_status.result()
+    print(ret.result_str)
+    thread_status = False
+
+
+def run():
+    global thread_status
+    channel = grpc.insecure_channel('0.0.0.0:50051')
+    stub = dronecore_pb2_grpc.DroneCoreRPCStub(channel)
+
+    mission_items = []
+    mission_items.append(dc.MissionItem(latitude=47.398170327054473,
```

Comment:
@julianoes Might be worth updating to more closely match our C++ example.

Reply:
Ok let me copy that over.